addon/modifiers/enable-input-autofocus.ts

```diff
@@ -0,0 +1,35 @@
+// @ts-ignore
+import { setModifierManager, capabilities } from '@ember/modifier';
+
+export default setModifierManager(
+  () => ({
+    capabilities: capabilities('3.22'),
+
+    createModifier() {
+      return {
+        element: null
+      };
+    },
+
+    installModifier(state: any, element: HTMLElement) {
+      state.element = element;
+      const localElement =
+        element.tagName.toLowerCase() === 'input'
+          ? (element as HTMLInputElement)
+          : (element.querySelector('input:not([disabled])') as HTMLInputElement);
+
+      if (!localElement.disabled) {
+        localElement.scrollIntoView({ behavior: 'smooth', block: 'center' });
+        localElement.focus({ preventScroll: true });
+      } else {
+        return;
+      }
+    },
+
+    destroyModifier() {
+      // We don't need to do anything here, but a function
```

Comment:
Nit: Don't know what others will think, but are those comments required ?

Reply:
Well we don't need 2 lines to say that it needs to remain blank but I'm not too keen on removing it either , but up to you 👍